docs/reference/Configuration.md

```diff
@@ -63,6 +63,23 @@ settings:
     # This value controls whether PHPUnit attempts to backup global variables
     # See https://phpunit.de/manual/current/en/appendixes.annotations.html#appendixes.annotations.backupGlobals
     backup_globals: true
+    
+    # PHPUnit can be strict about tests that do not test anything
+    # See https://phpunit.de/manual/current/en/risky-tests.html#risky-tests.useless-tests
+    reportUselessTests: false
+    
+    # PHPUnit can be strict about output during tests.
+    # See https://phpunit.de/manual/current/en/risky-tests.html#risky-tests.output-during-test-execution
+    disallowTestOutput: false
+    
+    # PHPUnit can be strict about tests that manipulate global state.
+    # See https://phpunit.de/manual/current/en/risky-tests.html#risky-tests.global-state-manipulation
+    beStrictAboutChangesToGlobalState: false
```

Comment:
tooooooooooooooooooooo long

what does it mean to be strict?
`no_global_changes`, `avoid_global_changes`,... 
naming can be improved